lib/src/components/NxGlobalSidebar/NxGlobalSidebarFooter.tsx

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+
+import React, { FunctionComponent } from 'react';
+import classnames from 'classnames';
+
+import NxFontAwesomeIcon from '../NxFontAwesomeIcon/NxFontAwesomeIcon';
+import NxTextLink from '../NxTextLink/NxTextLink';
+
+import { NxGlobalSidebarFooterProps as Props, nxGlobalSidebarFooterPropTypes } from './types';
+import { faQuestionCircle } from '@fortawesome/free-solid-svg-icons';
+export { NxGlobalSidebarFooterProps } from './types';
+
+const NxGlobalSidebarFooter: FunctionComponent<Props> = function NxNavigationSidebarLinks(props) {
+  const {
+    className,
+    supportText,
+    supportLink,
+    releaseText,
+    releaseNumber,
+    productTagLine,
+    showCreatedBy,
+    ...otherProps } = props;
+  const classes = classnames(className, 'nx-global-sidebar__footer');
+
+  return (
+    <footer className={classes} {...otherProps}>
+      { supportLink &&
+        <div className="nx-global-sidebar__support">
+          <NxTextLink href={supportLink || undefined}>
+            <NxFontAwesomeIcon icon={faQuestionCircle} />
+            <span className="nx-global-sidebar__support-text nx-global-sidebar__expanded-content">
+              {supportText || 'Help and Support'}
+            </span>
+          </NxTextLink>
+        </div>
+      }
+
+      { releaseText || releaseNumber &&
```

Comment:
When `releaseText` is present it gets rendered as a plain text node rather than rendering the div structure below. This can be seen in the first example in the gallery.


That this wasn't caught by a unit test brings my attention to the fact that this PR is missing unit tests!

Reply:
> When `releaseText` is present it gets rendered as a plain text node rather than rendering the div structure below. This can be seen in the first example in the gallery.
> 
> That this wasn't caught by a unit test brings my attention to the fact that this PR is missing unit tests!

Fixed the text here: https://github.com/sonatype/sonatype-react-shared-components/pull/387/commits/1fe5571bb3a04815a954540f315880cc109e4e3a